edgexfoundry/app-functions-sdk-go
# refactor: Use new Common Controller for handling common APIs

closes #1441







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

      <link to docs PR>

## Testing Instructions
Run non-secure EdgeX Stack
Build and Run App Template.
Send GET to localhost:59740/api/v3/ping
Send GET to localhost:59740/api/v3/version
Send GET to localhost:59740/api/v3/config
Verify above response match expected, i.e. same as current main branch
Send POST to localhost:59740/api/v3/secret with
```
{
   "apiVersion":"v3",
   "secretName":"credentials",
   "secretData":[
      {
         "Key" :"username",
         "Value":"user"
      },
      {
         "Key" :"password",
         "Value":"pwd"
      }
   ]
}
```
Verify 201 Created response code
Send GET to localhost:59740/api/v3/config again
Verify Writable.InsecureSecrets has the new `credentials` secret 


## New Dependency Instructions (If applicable)
